test/README.md

```diff
@@ -201,6 +201,25 @@ The following is a list of all available settings:
                          default: (none). Test files with this variable will be ignored unless
                          the D_OBJC environment variable is set to "1"
 
+    EXTRA_ARTIFACT:     artefact used in the test (configuration) which needs to be generated
+                        by invoking a specified command before running the test.
+
+                        Syntax: <artifact> => <command>
+
+                        Both <artifact> and <command> can use bash-like variables as defined below
+                        to handle platform specific build whereas <command> also can use $@ to
+                        refer to <artifact>. The <command> will be executed using `std.process`.
```

Comment:
Not sure whether the last sentence is necessary for a summary documentation. That's more a technical detail.